### PR TITLE
[eclipse/xtext#1633] Allow to redirect compile folders to bin, bin-test with m2e

### PR DESCRIPTION
**Pull request by szarnekow**
see eclipse/xtext#1633

### CODE REVIEW

**Comment by szarnekow** [comment]
TODO: update oomph setup to set the preference, update all projects (.classpath and build.properties)

File: org.eclipse.xtext.m2e/src/org/eclipse/xtext/m2e/BinFolderConfigurator.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+						IClasspathEntry copy = JavaCore.newSourceEntry(entry.getPath(), entry.getInclusionPatterns(), entry.getExclusionPatterns(), binPath, entry.getExtraAttributes());
+						rawClasspath[i] = copy;
+					}
+				}
+			}
+			javaProject.setRawClasspath(rawClasspath, monitor);
+		}
+	}
+
+	/**
```
Comment (ArneDeutsch) [inline org.eclipse.xtext.m2e/src/org/eclipse/xtext/m2e/BinFolderConfigurator.java]: Delete or elaborate comment.

Comment (szarnekow) [reply org.eclipse.xtext.m2e/src/org/eclipse/xtext/m2e/BinFolderConfigurator.java]: :+1:

Comment (szarnekow) [reply org.eclipse.xtext.m2e/src/org/eclipse/xtext/m2e/BinFolderConfigurator.java]: Done.

**Pull request merged**